docs/resource_management/RESOURCE_SCALING.md

```diff
@@ -0,0 +1,39 @@
+# Resource requests and limits
+To ensure that Kubernetes appropriately schedules resources, the respective product `values.yaml` are configured with default `cpu` and `memory` [resource request values](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/).
+
+## Resource requests
+The default resource request that are used for each product are defined below. Using the [formula](#Memory-request-sizing), the `memory` specific values are derived from the default `JVM` requirements defined for each product's Docker container.
```

Comment:
I wonder how these values were set in the first place. I like the approach that we are consistent I am just not sure how these values were set in the first place.

Reply:
Spoke with Dave about this. These values were actually supplied by the product teams.

Reply:
the respective product `values.yaml` is configured